tests/scripts/tests-setup.sh

```diff
@@ -3,6 +3,16 @@
 
 set -x
 
+echo "Setting Host Network Backend as netavark..."
+sudo cp /usr/share/containers/containers.conf /etc/containers/containers.conf
+sudo sed -i '/#network_backend=*/c\network_backend="netavark"' /etc/containers/containers.conf
+sudo podman system reset --force
```

Comment:
@rhatdan could you please confirm this is required? This is a fresh systes, if it's not required we can remove.

Reply:
Yes, if you start on a fresh system you can remove the system reset command.